### PR TITLE
Added /blockchain/insert docs

### PR DESCRIPTION
**Pull request by waddahAldrobi**
This is a start, I've labelled it as (beta) for now.
We can iterate from here

### CODE REVIEW

File: api-reference/tables/tables-openapi.json [hunk 1]
```diff
@@ -309,6 +309,103 @@
...
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/InsertTableResponse"
```
Comment (vegarsti) [inline api-reference/tables/tables-openapi.json:354]: Nice!

**Review by vegarsti — APPROVED** [review]
Thanks!

File: mint.json [hunk 2]
```diff
@@ -771,6 +771,10 @@
     {
       "group": "General API Endpoints",
       "pages": [
+        {
+          "group": "Blockchains",
```
Comment (vegarsti) [inline mint.json:775]: Good!

**Review by hawkaa — COMMENTED** [review]
Fantastic! Did we consider adding any "call to action" for getting in touch with sales, or mention something about the plans/pricing? I'm not sure what exactly to write, but we should probably point people in the right direction.

**Comment by waddahAldrobi** [comment]
> Fantastic! Did we consider adding any "call to action" for getting in touch with sales, or mention something about the plans/pricing? I'm not sure what exactly to write, but we should probably point people in the right direction.

Jackie added it in the overview of Blockchains 
https://docs.dune.com/api-reference/tables/blockchain/overview

**Comment by agaperste** [comment]
yes, just edited a bit more to make it more obvious
[image: image.png]

**Pull request merged**